fury/tests/test_actors.py

```diff
@@ -869,6 +869,56 @@ def test_points(interactive=False):
     npt.assert_equal(report.objects, 3)
 
 
+def test_dotted_line(interactive=False):
+    start_pos = np.array([[0, 0, 0]])
+    end_pos = np.array([[1, 1, 1]])
+    radius = 0.01
+    num_points = 10
```

Comment:
We can have two options here. 1. define the num_points directly. 2. or the length of segments and gaps are fixed and based on the length of line, the number of segments with gaps will increase properly depending on length of the line. 
For option 2, the size of segment and size of the gap (between segments) are fixed, so if we increase the length of line, we are just adding more segments with gap (with fixed size). The size of last segment that reaches the end_pos, can be flexible until it reaches the end_point. I sent you a GIF file for a better explanation. 
![dashed_line](https://user-images.githubusercontent.com/32108826/114602919-20bfc200-9c65-11eb-86bd-8ef9cf238096.gif)

Reply:
Can you have a look at the line fraction parameter of dashed lines actor? It allows the developer to change the length (or space) without altering the number of segments.

Reply:
The line_fraction parameter allows the user to alter the spaces without changing the total number of segments, as shown below:

1. The dashed line with two segments with line fraction set to 25% (0.25)
![25_percent](https://user-images.githubusercontent.com/56040092/116513063-a5a60f00-a8e6-11eb-83ee-cfd74d8ebc91.png)

2. The dashed line with two segments with line fraction set to 50% (0.50)
![50_percent](https://user-images.githubusercontent.com/56040092/116513163-cb331880-a8e6-11eb-99cd-ff82de19ba21.png)

3. The dashed line with two segments with line fraction set to 75% (0.75)
![75_percent](https://user-images.githubusercontent.com/56040092/116513179-cff7cc80-a8e6-11eb-9c3a-331d14b83d72.png)

The number of segments is constant (two), but the spacing can be varied.